pkg/controller/constrainttemplate/constrainttemplate_controller_test.go

```diff
@@ -421,6 +393,32 @@ func TestReconcile(t *testing.T) {
 		}
 	})
 
+	t.Run("VapBinding should be created", func(t *testing.T) {
+		suffix := "VapBindingShouldBeCreated"
+		logger.Info("Running test: VapBinding should be created")
+		constraint.DefaultGenerateVAPB = ptr.To[bool](true)
```

Comment:
we need a test case for VapBinding should not be created when DefaultGenerateVAPB is true and CT is just rego